packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -0,0 +1,269 @@
+import {helpers, ShopperLogin, ShopperLoginTypes} from 'commerce-sdk-isomorphic'
+import jwtDecode from 'jwt-decode'
+import {ApiClientConfigParams} from '../hooks/types'
+import {BaseStorage, LocalStorage, CookieStorage} from './storage'
+
+type Helpers = typeof helpers
+interface AuthConfig {
+    shortCode: string
+    organizationId: string
+    clientId: string
+    siteId: string
+    redirectURI: string
+    proxy: string
+}
+
+interface JWTHeaders {
```

Comment:
Dumb question, how do we know the types of these headers? Is there I resource I can take a look?

Reply:
i looked at the network response on browser...